setup.cfg

```diff
@@ -30,8 +30,10 @@ extend-ignore =
     E203, # Incompatible with black see https://github.com/psf/black/issues/315
     W503, # Incompatible with black
     B901 # Combine yield and return statements in one function
-max-complexity = 83
-max-line-length = 138
+max-complexity = 36
+max-line-length = 120
```

Comment:
Can we get this lower? Or is this a sane current max?

Reply:
There's a lot of over 100+, let's try 110.